tests/testthat/test-control_chart.R

```diff
@@ -0,0 +1,69 @@
+context("Checking control_chart")
+
+# Setup ------------------------------------------------------------------------
+library(ggplot2)
+test_df <- tibble::tibble(
+  var1 = rep(letters[1:2], each = 10),
+  var2 = rep(letters[2:5], times = 5),
+  outcome = c(1:19, 25)  # To get different mean and median
+)
+
```

Comment:
Test if grouping variables are used, does it make lattice plots correctly?

Reply:
That could be done, but it requires disembowling the ggplot object. And this is the end of a pipe -- nothing else breaks if this does. I'm open to discussion on this, but I think we can probably 80/20 our testing effort toward stuff that would have downstream consequences if it misbehaves. Thoughts @mmastand @Aylr ?